### ISSUE TITLE
Pathfinding ensure works with +ve and -ve latlons

### ISSUE THREAD

**Issue by tylerlum** [UBCSailbot/global-pathfinding#37]
Longitudes can either be in [0, 360) or [-180, 180). Our code should be able to handle both. Make sure that the path and the wind go in the "right direction' (not on the other side of the earth).

Currently negative numbers do not work because of argument syntax. Need to fix this to ensure it works.

Try out all permutations of:

```
./build/bin/pathfinder_cli -p 8 --navigate 48 235 21 203
./build/bin/pathfinder_cli -p 8 --navigate 48 -125 21 203
./build/bin/pathfinder_cli -p 8 --navigate 48 235 21 -157
./build/bin/pathfinder_cli -p 8 --navigate 48 -125 21 -157
```

Make sure that pathfinding does not segfault in all situations. Make sure they all make the same path. Make sure that the wind.kml received is the same each time (not on other side of the world)

**Comment by tylerlum** [comment]
More specific task: make sure this works for whatever the satellite could possibly give us, never segfault with valid inputs.

**Comment by tylerlum** [comment]
This will be addressed by Kieran's bash script that will ensure it is always passed +ve latons. Will close this issue when that PR is merged in.

**Comment by tylerlum** [comment]
Addressed by #49 

```
    if (pointToPrint == countPoints) {      latToPrint = lat_str;      lonToPrint = std::stod(lon_str) &lt; 0 ? std::to_string(360 + std::stod(lon_str)) : lon_str;    }
```

Bash script start with +ve values and this change ensures all output values are positive as well. As long as we keep all positives at all times, this issue will not arise.
